### PR TITLE
Add an icon in themeswitcher to choose user default theme

### PR DESCRIPTION
**Pull request by gwenandres**
Hello, 
In this PR I propose to add a star icon (new.svg icon) on themes in themeswitcher to visualize and change default theme (for user logged).

Screenshot 2024-01-30 at 12-30-11 QGIS Web Client 2

I add an option (like in settings plugin) in themeswitcher plugin config to get or not this icon. 

What do you think about that ? 

Have a good day

### CODE REVIEW

File: components/ThemeList.jsx [hunk 1]
```diff
@@ -85,6 +89,9 @@ class ThemeList extends React.Component {
         const addLayersTitle = LocaleUtils.tr("themeswitcher.addlayerstotheme");
         const addTitle = LocaleUtils.tr("themeswitcher.addtotheme");
         const openTabTitle = LocaleUtils.tr("themeswitcher.openintab");
+        const changeDefaultUrlTitle = LocaleUtils.tr("themeswitcher.changedefaulttheme");
+        const username = ConfigUtils.getConfigProp("username");
+
```
Comment (manisandro) [inline components/ThemeList.jsx:94]: Please sort props alphabetically (resp. configure eslint in your editor).

**Comment by manisandro** [comment]
Thanks, looks like a nice improvement

File: components/ThemeList.jsx (hunk 1 above)
Comment (gwenandres) [reply components/ThemeList.jsx:95]: Sorry for that !

**Comment by manisandro** [comment]
Thanks!

**Pull request merged**